app/cdash/app/Model/Build.php

```diff
@@ -992,12 +986,11 @@ private function GetTests(string $criteria, int $maxitems = 0): array|false
         }
 
         $sql = "
-            SELECT t.name, b2t.id AS buildtestid, b2t.details
-            FROM test t
-            JOIN build2test b2t ON t.id = b2t.testid
+            SELECT b2t.testname AS name, b2t.id AS buildtestid, b2t.details
+            FROM build2test b2t
             WHERE b2t.buildid = :buildid
             AND $criteria
-            ORDER BY t.id
+            ORDER BY b2t.testname
```

Comment:
@zackgalbreath  This change is the reason the test name change was necessary.  Since it is no longer (easily) possible to order test names in sequential order, I changed the ordering to instead order by test name in lexicographical order.  It's possible to restore the original functionality if needed, but doing so would make the query slower.  I was unable to find anywhere where changing the ordering would break anything.